src/json/json-document.ts

```diff
@@ -54,4 +54,8 @@ export class ParsedJsonDocument extends ParsedDocument<Json, Visitor> {
   sourceRangeForNode(_node: Json): SourceRange {
     throw new Error('Not Implemented.');
   }
+
+  stringify() {
+    return JSON.stringify(this.ast, null, 2);
```

Comment:
FYI, I found this JSON parser that can report source positions: https://github.com/dscape/clarinet

Would need some work to use...